app/ui/omni-box.js

```diff
@@ -32,10 +32,18 @@ class OmniBox extends HTMLElement {
     this.targetUrl = this.$('.omni-box-target-input')
 
     this.input.addEventListener('focus', () => {
-      this.input.showInput()
```

Comment:
I don't think this line does anything since input should already be visible if it can be focused?
Also generates error in console. Think it should be `this.showInput()`.

Reply:
I think this is needed for the comment mentioned here: https://github.com/AgregoreWeb/agregore-browser/commit/5d4dae271b9c95cede9a8c03b491c96dbb2b23cc

I guess Electron can be a bit buggy. 😅